explorer/src/entity/ChainlinkNode.ts

```diff
@@ -16,9 +16,18 @@ export interface IChainlinkNodePresenter {
 
 @Entity()
 export class ChainlinkNode {
-  public static build(name: string, secret: string): ChainlinkNode {
+  public static build({
+    name,
+    url,
+    secret
+  }: {
+    name: string
+    url?: string
+    secret: string
+  }): ChainlinkNode {
     const cl = new ChainlinkNode()
     cl.name = name
+    cl.url = url || ''
```

Comment:
How do you feel about making this a conditional so that empty string results in a `NULL` column value along the lines of the comment above ^?